### ISSUE TITLE
Improve Accessibility Standards

### ISSUE THREAD

**Issue by birtony** [Seneca-CDOT/telescope#1608]
**What would you like to be added**:
- We should improve accessibility standards on Telescope to comply with WCAG: https://www.w3.org/TR/WCAG21/
- Here is a link to React documentation regarding WCAG standards: https://reactjs.org/docs/accessibility.html
- We could also use Google Lighthouse and [Webhint](https://webhint.io/) to test our website manually or https://github.com/dequelabs/axe-core-npm/tree/develop/packages/react to integrate automated tests into our code base

**Why would you like this to be added**:
It would be nice to make sure that out website is accessible for everyone. Complying with website accessibility standards is also a requirement from the Government of Ontario: https://www.ontario.ca/page/how-make-websites-accessible

**Comment by izhuravlev** [comment]
cc @izhuravlev, @rjayroso, 
We need to file a number of issues and list them there. 

The issues should describe what exactly do we want to do to "Improve Accessibility" - small actions, small problems, that overall create a big change.

**Comment by yuanLeeMidori** [comment]
Update:

This issue has been addressed with multiple sub-issues and PRs (see the [label of accessibility](https://github.com/Seneca-CDOT/telescope/issues?q=label%3Aaccessibility+))

There is more we can do in the future.

**Comment by joelazwar** [comment]
![image](https://user-images.githubusercontent.com/43277890/152995166-7fcdf06d-ec47-47af-8ffd-7c91f2dc65ac.png)

Some accessibility stuff that lighthouse detected.

**Comment by humphd** [comment]
I don't think we can solve missing `alt` for `img` elements that are stored in the blog HTML.  Same goes for any other errors related to post content.

However, we can fix our own UI to have proper names/labels, etc.  We should focus on that.
